var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -87,6 +87,7 @@ class Gromacs(CMakePackage, CudaPackage):
         description="Produces a double precision version of the executables",
     )
     variant("plumed", default=False, description="Enable PLUMED support")
+    variant("cufftmp", default=False, description="Enable Multi GPU FFT support")
```

Comment:
If the semantic is the one discussed in the comments, then:
```suggestion
    variant("cufftmp", default=False, when="+cuda+mpi", description="Enable Multi GPU FFT support")
```
This means that the variant exists only when `+cuda+mpi` are both active.

Reply:
But do you say it is good this way, or bad?

Reply:
Whenever possible it's better to have conditional variants than conflicts (especially conflicts with default values) due to how the concretizer optimizes to reach the "best" possible solution.